### PR TITLE
Update long-format documentation

### PR DESCRIPTION
**Pull request by ahasverus**
This PR is linked to #45

### CODE REVIEW

File: vignettes/long-format.Rmd [hunk 1]
```diff
@@ -9,51 +9,124 @@ vignette: >
...
 
-For example, the raw data used to generate the internal data of `funbiogeo`:
+The `funbiogeo`package requires that information is structured in three different datasets:
+
+- the **species x traits** `data.frame` (`species_traits` in `funbiogeo`)
+- the **site x species** `data.frame` (`site_species` in `funbiogeo`)
+- the **site x locations** object (`site_locations` in `funbiogeo`)
+
+
+
```
Comment (Rekyt) [inline vignettes/long-format.Rmd:32]: Just a question: you always seem to add so much space between paragraphs. You don't like having single empty lines or you prefer to have more space in the final document?
(Real question not a suggestion😅)

File: vignettes/long-format.Rmd [hunk 2]
```diff
@@ -9,51 +9,124 @@ vignette: >
...
+## Wide vs long format
+
+
+
+In `funbiogeo` these three datasets **must be** in a wide format (variables are 
+spread across columns), but sometimes information is structured in a long format 
+(one column for several variables).
```
Comment (Rekyt) [inline vignettes/long-format.Rmd:39]:
```suggestion
In `funbiogeo` these three datasets **must be** in a wide format (=where one row hosts several variables across columns), but sometimes information is structured in a long format 
(=one observation per row, also called [**tidy** format](https://r4ds.had.co.nz/tidy-data.html)).
```

File: vignettes/long-format.Rmd [hunk 3]
```diff
@@ -9,51 +9,124 @@ vignette: >
...
+
+All these functions take a long dataset as input (argument `data`), where one 
+row corresponds to the occurrence/abundance/coverage of one species at one site 
+and output a wider object.
+
+
+
+## Usage
+
+
```
Comment (Rekyt) [inline vignettes/long-format.Rmd:97]:
```suggestion
`funbiogeo` provides a small excerpt of long format data to show how to use the functions.
This data sits at `system.file("extdata", "raw_mammals_data.csv", package = "funbiogeo")`
```

**Review by Rekyt — COMMENTED** [review]
Apart from the few changes, the vignette seems ready to me!
Great work! 👍

File: vignettes/long-format.Rmd (hunk 1 above)
Comment (ahasverus) [reply vignettes/long-format.Rmd:32]: Yes 😄  I like that my code breathe 😄 othewise I suffocate 😄

**Pull request merged**